python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -157,8 +157,15 @@ def measure(self, measRecord, exposure):
 
         ra, dec = self.computeRaDec(exposure, xc, yc)
 
+        if measRecord.getShapeFlag():
+            self.log.warning("Shape flag is set for measRecord: %s. Trail measurement "
+                             "will not be made.", measRecord.getId())
+            self.flagHandler.setValue(measRecord, self.FAILURE.number, True)
```

Comment:
You still need to set a flag for the reason for the failure. You'd had a `SECOND_MOMENT` flag before, and we want that set here.

Reply:
Flag `self.SHAPE` has been added to indicate the shape flag existed so the trail was not measured.